lang/rust/Makefile

```
@@ -0,0 +1,4 @@
+all: helloworld
+
+helloworld: helloworld.rs:
```

Without removing the trailing colon I get

```
$ make
Makefile:3: *** target pattern contains no '%'.  Stop.
```

```suggestion
helloworld: helloworld.rs
```